support-workers/package.json

```diff
@@ -6,13 +6,15 @@
   "scripts": {
     "test": "jest --group=-integration",
     "it-test": "jest --group=integration",
-    "lint": "eslint src/typescript/**/*.ts",
+    "lint:check": "eslint src/typescript/**/*.ts",
+    "lint:fix": "eslint --fix src/typescript/**/*.ts",
     "check-formatting": "prettier --check **.ts",
     "build": "tsc",
+    "bundle": "esbuild src/typescript/lambdas/*.ts --bundle --platform=node --target=node22 --outdir=target/typescript",
     "clean-target": "rm -rf target && mkdir -p target",
-    "install-runtime-dependencies": "pnpm install --ignore-scripts --modules-dir target/typescript/node_modules --production",
-    "build-and-zip": "pnpm clean-target && pnpm build && pnpm install-runtime-dependencies && cd target/typescript && zip -r support-workers.zip ./* -x \".*\" -x \"__MACOSX\"",
-    "package": "pnpm check-formatting && pnpm lint && pnpm test && pnpm build-and-zip"
+    "zip": "cd target/typescript && zip -r support-workers.zip ./* -x \".*\" -x \"__MACOSX\"",
```

Comment:
is there a standard for what these commands should be?

in support service-lambdas we use type-check, lint or lint --fix and check-formatting or fix-formatting
Obviously I'm not working on support-workers much now, but it is already a bit of a pain to remember the commands so it would be nice if they could be consistent across projects